web/views/tracker_view.ex

```diff
@@ -112,7 +112,7 @@ defmodule Healthlocker.TrackerView do
                   |> Map.get(:hours_slept)
       List.insert_at(list, 0, hours_slept)
     else
-      List.insert_at(list, 0, 0)
+      List.insert_at(list, 0, "null")
```

Comment:
@RobStallion yeah, I _understood_ that. But `null` values are _Bad UX_ and we should "_push back_". Our objective is help "_steer_" the users towards desirable behaviour/actions. Having a "_empty_" column on the graph is just a `void`, having a `0` reminds the person that they did not input/capture any data for that day. 
I would _personally_ take this UX a step _further_ by putting some text in the empty datapoint asking the person how they were feeling on the day that they did not enter any data to _understand_ `Why` they didn't capture any "_progress_" toward their goal. 

But ... In the interest of _shipping_ this and letting _users_ give feedback, 
let's just do _exactly_ what has been requested and _ignore_ UX. 🙈 🙊  👍